docs/installation/INSTALL_LINUX.md

```diff
@@ -59,17 +59,15 @@ This will create InvokeAI folder where you will follow the rest of the steps.
 (invokeai) ~/InvokeAI$ python3 scripts/preload_models.py
 ```
 
-    !!! note
```

Comment:
@lstein This syntax is used by the job which transforms the MD into the HTML for Pages

Reply:
Sure, will put back.

How does this reorganization of the requirements files interact with your one-click installer script. And how is that going?

Reply:
> interact with your one-click installer script

All of the requirements and environment files will be deleted

> how is that going?

"Works For Me" - but I think we all know how dangerous that is :rofl: 

TL;DR - When I feel confident enough, I'm going to open a draft PR. At that point, I will need testing on as large a platform (platform == OS/CPU/(GPU|!GPU) matrix as humanly possible, along with as much of the preflight checks as people have patience for.

Also of note: I don't think that this will be the final stop on this particular subway - that would be a stand-alone Python application per-platform - but Python application bundling is dark magic, and AFAICT there's no ***standard*** (Conda bundles or whatever they're called *are not* a "standard")